proto/v1/discovery.proto

```diff
@@ -80,4 +80,7 @@ message DiscoveryOccurrence {
   
   // The last time this resource was scanned.
   google.protobuf.Timestamp last_scan_time = 5;
+
+  // Output only. The time occurrences related to this discovery occurrence were archived.
```

Comment:
Use field behavior OUTPUT_ONLY instead of the comment. https://google.aip.dev/203#output-only

See https://github.com/grafeas/grafeas/blob/master/proto/v1beta1/spdx.proto#L271

Reply:
done